src/app/help/help.component.html

```diff
@@ -1,23 +1,62 @@
-<ion-header>
-  <ion-toolbar>
-    <ion-title>
-      Help
-    </ion-title>
-  </ion-toolbar>
-</ion-header>
+<ion-app title="Settings" class="jsmbp-settings-container">
+	<ion-content padding class="has-header">
+      <h1 text-center>Settings</h1>
+		<ion-card>
+			<ion-item class="item-divider mb-blue">
+				My Account
+				<p ng-if="email">
+					<label>Email: </label>
+					<span ng-bind="email"></span>
+				</p>
+			</ion-item>
 
-<ion-content>
-  <ion-list>
-    <ion-list-header>
-      <ion-label>
-        Follow us on Twitter
-      </ion-label>
-    </ion-list-header>
-    <ion-item>
-      <ion-label>
-        <ion-icon name="logo-ionic" slot="start"></ion-icon>
-        @ionicframework
-      </ion-label>
-    </ion-item>
-  </ion-list>
-</ion-content>
\ No newline at end of file
+			<ion-item class="row">
+				<label class="item item-input item-select">
+					<div class="input-label">Select Country</div>
+					<select name="country" id="country" ng-model="countryModel" ng-change="updateCountry(countryModel)">
+						<option ng-repeat="countryCode in countryCodes" value="{{countryCode.code}}">{{countryCode.name}}</option>
```

Comment:
`ng-repeat` is Angular 1 syntax as well, use `*ngFor` instead.
Also, need to define `countryCodes` in `.component.ts`